system_tests/lewis_emulators/stanford_sr400_photon_counter/device.py

```diff
@@ -34,18 +34,18 @@ def set_random_status(self) -> None:
 
     def get_count_a(self) -> int:
         if self.counting:
-            self.count_a += random.randint(10, 15)
+            self.count_a = 20
         return self.count_a
 
     def get_count_b(self) -> int:
         if self.counting:
-            self.count_b += random.randint(10, 15)
+            self.count_b = 15
```

Comment:
Do we actually need this any more? If we need to change the counter values we can do that via the emulator backdoor rather than hard-coding values here. But I can't see that this 15/20 are being used in the tests anyway...

Reply:
This is invoked by the interface. The value is set to a non-zero value and verified in the test that a non-zero value is returned after the start/stop is performed, the initial value being 0.

Reply:
ok - I think it would be _better_ to use the emulator backdoor for this, but won't hold up the review for it.